src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -411,6 +411,13 @@ public function createTable(): Table
         return (new Table($output))->setStyle($style);
     }
 
+    public function hr(string $word = '─', string $format = '%s'): void
+    {
+        $line_width = (new Terminal())->getWidth() ?: self::MAX_LINE_LENGTH;
+        $line = str_repeat($word, $line_width / mb_strlen($word));
```

Comment:
Should use `Helper::width` instead of mb_strlen
also, the second argument can be a float currently, this looks suspicious